joomla/joomla-cms
# [4.0] Field media custom element clean up

Pull Request for Issue # .

### Summary of Changes
- Eslint
- Proper event listeners


### Testing Instructions
Create a new article and check the image in tinyMCE and also the intro/fulltext image field

### Expected result



### Actual result



### Documentation Changes Required

Nope
